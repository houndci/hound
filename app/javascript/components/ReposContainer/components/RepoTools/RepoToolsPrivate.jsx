import React from 'react';

export default class RepoToolsPrivate extends React.Component {
  render() {
    const { onPrivateClicked } = this.props;

    return (
      <div className="repo-tools-private">
        <form method="post" action="/auth/github?access=full">
          <button
            className="repo-tools-private-button"
            type="submit"
            onClick={onPrivateClicked}
          >
            <span>Include private repos</span>
          </button>
        </form>
      </div>
    );
  }
}
