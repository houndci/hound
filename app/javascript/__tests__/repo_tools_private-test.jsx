import RepoToolsPrivate from '../components/ReposContainer/components/RepoTools/RepoToolsPrivate';

it('renders appropriately', () => {
  const wrapper = shallow(
    <RepoToolsPrivate />
  );
  expect(wrapper).toMatchSnapshot();
});
