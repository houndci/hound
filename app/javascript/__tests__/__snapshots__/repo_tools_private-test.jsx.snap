// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders appropriately 1`] = `
<div
  className="repo-tools-private"
>
  <form
    action="/auth/github?access=full"
    method="post"
  >
    <button
      className="repo-tools-private-button"
      type="submit"
    >
      <span>
        Include private repos
      </span>
    </button>
  </form>
</div>
`;
